backend/django_core/apps/user/validators/username.py

```diff
@@ -12,3 +12,7 @@
 def username_validator(username: str) -> None:
     if username in INVALID_USERNAMES:
         raise ValidationError(f"Username {username} not allowed")
+
+    # It can raise issues with usernames like `baseplate-admin#0001#0001`
+    if "#" in username:
```

Comment:
dont raise when username is like `baseplate-admin#0001` ?
this has `#` symbol

Reply:
> dont raise when username is like baseplate-admin#0001 ?
> this has # symbol

That wont be a problem due to how we have implemented user discriminator lookup.

Check : https://github.com/baseplate-admin/CoreProject/blob/0104754a0e8eac1a77637b444f954d9754cc7f2a/backend/django_core/mixins/manager/username_with_discriminator.py#L19-L33

Reply:
Okay now makes sense lol